common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3219,12 +3219,63 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_TABLE_OPERATION" : {
+    "message" : [
+      "Table <tableName> does not support <operation>."
+    ],
+    "subClass" : {
+      "WITHOUT_SUGGESTION" : {
+        "message" : [
+          ""
+        ]
+      },
+      "WITH_SUGGESTION" : {
+        "message" : [
+          "<suggestion>."
```

Comment:
Besides of the hits, how many others exist in the source code?
```scala
  private def alterViewTypeMismatchHint: Option[String] = Some("Please use ALTER TABLE instead.")

  private def alterTableTypeMismatchHint: Option[String] = Some("Please use ALTER VIEW instead.")
```

Can you move the text to `error-classes.json`?

Reply:
Done